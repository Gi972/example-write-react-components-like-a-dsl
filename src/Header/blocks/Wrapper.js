import React from 'react'
import './Wrapper.css'

export default function Wrapper({ children }) {
  return(
    <header className="App-header">
      {children}
    </header>
  )
}
