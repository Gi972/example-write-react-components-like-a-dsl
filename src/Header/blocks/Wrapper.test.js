import React from 'react'
import renderer from 'react-test-renderer'

import Wrapper from './Wrapper'

describe('<Wrapper></Wrapper>', () => {
  it('renders', () => {
    expect(<Wrapper>child</Wrapper>).toMatchSnapshot()
  })
})
