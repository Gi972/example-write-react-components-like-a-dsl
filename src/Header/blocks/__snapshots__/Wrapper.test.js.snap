// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper></Wrapper> renders 1`] = `
<Wrapper>
  child
</Wrapper>
`;
